components/count-badge/README.md

```diff
@@ -0,0 +1,30 @@
+# Count Badge
+
+## d2l-count-badge
+
+The `d2l-count-badge` element is a web component to display a number count, either in a `"notification"` or `"count"` style.
+
+
+![Notification Badge](./screenshots/count-badge-notification-small.png?raw=true)
+![Count Badge](./screenshots/count-badge-count-large.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/count-badge/count-badge.js';
+</script>
+<d2l-count-badge badge-size="small" badge-type="notification" number="100"></d2l-count-badge>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `number` | Number, required | The number to display on the badge. |
+| `size`, default: `small` | String | The size of the badge. Valid options are `"small"` and `"large"`. |
+| `type`, default: `count` | String | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and truncate numbers above 99 to `99+`, while count badges are grey and do not truncate the number. |
+| `max-digits`, default: `no limit` | Number | For `"count"` type badge, optionally specify a maximum number of digits, after which numbers are truncated. |

```

Comment:
I don't love having attributes that only do something if another attribute is set to a certain value... usually it's an indicator that the component could maybe be broken up into two.

For `max-digits` though, the `notification` type also kind-of supports it in that it truncates 999 to "99+"... so I'm wondering if maybe we could support it and it just defaults to `2` when the type is `notification` and nothing otherwise. That's also a little weird though.

Reply:
That sounds like a good idea to me, we can default the notification type to 2 and make it overridable. I can add a descriptive comment in the readme so hopefully its not too confusing.

Reply:
Jeff had mentioned "...I think Notifications should force the 2 digit limit if that’s possible without hackery/awkwardness in the code"

Reply:
Oh, but then also said "If it’s annoying to implement that way, then yeah, it can just be the default for notifications and consumers can override it"